editor/animation_track_editor.cpp

```diff
@@ -6265,7 +6265,7 @@ AnimationTrackEditor::AnimationTrackEditor() {
 	cleanup_all->set_text(TTR("Clean-up all animations"));
 	cleanup_vb->add_child(cleanup_all);
 
-	cleanup_dialog->set_title(TTR("Clean-Up Animation(s) (NO UNDO!)"));
+	cleanup_dialog->set_title(TTR("Clean-Up Animation(s) (NO UNDO)"));
```

Comment:
Should it be in the text instead of title, like in other cases?

Reply:
Yeah, title isn't very readable...
![image](https://user-images.githubusercontent.com/2223172/148814620-1b850a23-58eb-46fe-94ed-b2da38ba5bf5.png)